docs/en/05_Contributing/04_Release_Process.md

```diff
@@ -33,34 +36,98 @@ announced on the ["releases" forum category](https://forum.silverstripe.org/c/re
 
 ## Release numbering {#release-numbering}
 
-SilverStripe follows [Semantic Versioning](http://semver.org).
+Silverstripe CMS follows [Semantic Versioning](http://semver.org).
 
-All SilverStripe modules (including silverstripe/framework) may have patch releases created at any time, and will
+All Silverstripe CMS modules (including `silverstripe/framework`) may have patch releases created at any time, and will
 not necessarily match other core module patch release numbers. For example, your project may be using
-silverstripe/cms 4.3.1 with silverstripe/versioned 4.3.9.
+`silverstripe/cms` 4.3.1 with `silverstripe/versioned` 4.3.9.
 
-All SilverStripe recipes are released in lock step with each other. For example, silverstripe/installer 4.3.1 will
-contain silverstripe/recipe-cms 4.3.1 and silverstripe/recipe-core 4.3.1. These recipes may contain various patch
+All Silverstripe CMS recipes are released in lock step with each other. For example, `silverstripe/installer` 4.3.1 will
+contain `silverstripe/recipe-cms` 4.3.1 and `silverstripe/recipe-core` 4.3.1. These recipes may contain various patch
 versions of its modules within the same minor release line (4.3 in this example).
 
+## Regular quarterly releases
+
+A regular Silverstripe CMS minor release is tagged every quarter. This involves creating a new minor increment of 
+each core recipe. Regular quarterly releases are preceded by pre-stable releases to help lay the groundwork for the
+stable release.
+
+### Beta phase
+
+At the start of the Beta phase:
+* new minor branches for core modules are forked from the default branches
+* a feature freeze is instituted on the minor branches
+* a `beta1` release is tagged for recipes and related modules.
+
+The beta phase lasts approximately one month. Additional betas may be released to address critical issues holding back
```

Comment:
```suggestion
While the beta phase lasts additional betas may be released to address found issues
```

We should avoid mentioning any timeframes in the documentation since that's a subject for change.

Reply:
I think the timeframe is kind of important here. If I'm telling you "I will tag an RC a month from now", that gives you a good indication of how much time you have to put your feedback in. If we don't specify a time frame it's not as clear.

It doesn't stop us from changing it again in a few release if we decide it's not practical.

Reply:
we can give expected time frames when we announce releases. it doesn't have to be carved in stone. E.g. when we do a feature freeze we can mention in the announcement itself how much time we expect it to last until we do the RC

Reply:
I agree with @dnsl48 here. I've made a different suggestion to outline that we will communicate timelines at the time of tagging the beta.